### DISCUSSION TITLE
Is there any reason that files would be missing major hospitals?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#553] [General]
In some markets that we have looked at closely, some of the major hospitals are not included in the in_network files. These hospitals list the plans on the hospital's page for insurance plans that are accepted.

Is there any reason that files would be missing major hospitals? I realize that there are many separate in_network files.

Is anyone else seeing insurers leaving out major hospitals?

**Comment by tjmc** [comment]
Could you give an example?

**Reply by cheernurse** [reply]
Hey, have you guys made a list of downloadable URLs? 
Can we share? 

My team just started to looking into some files. 
We also want to make auto-update-check, since they are saying update is possible every 30 days.

**Reply by santaviccap** [reply]
My organization is seeing the same thing.  Multiple providers missing from INN files which are known to be in that network, as well as large swaths of procedure codes known to be covered.

**Comment by jltang-kp** [comment]
I would imagine the reason why is if payors could not express their compensation to providers in the given INN format and were not able to use the Free text field in time to express certain procedure codes.

Some payors have said in their disclaimers they have not been able to use the free text field, but not all.

**Reply by nmayle** [reply]
Why wouldn't they be able to use the free text field ("additional_information") ?  That field is in the 1.0 schema.
What types of payment arrangements can't be described in the current schema?  Is the issue payment arrangements or multiple codes defining a rate e.g. CPT and revenue code?
